Source/JavaScriptCore/inspector/remote/RemoteInspector.cpp

```diff
@@ -227,11 +221,14 @@ TargetListing RemoteInspector::listingForTarget(const RemoteControllableTarget&
 
 void RemoteInspector::updateTargetListing(TargetID targetIdentifier)
 {
-    auto target = m_targetMap.get(targetIdentifier);
-    if (!target)
+    auto findResult = m_targetMap.find(targetIdentifier);
+    if (findResult == m_targetMap.end())
+        return;
+
+    auto target = findResult->value;
```

Comment:
`auto` -> `RefPtr`?

Reply:
But also, what was wrong with the existing code? Why do we have to switch to iterators. Seems like `get()` would return a `RefPtr` to you, which would be fine.